ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -383,28 +401,21 @@ gcp_file_open(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name
     return 0;
 }
 
+// POSIX remove is not supported for cloud objects.
 static int
 gcp_remove(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, uint32_t flags)
```

Comment:
Do we need to apply a [[maybe_unused]] tag here or `WT_UNUSED` here?

Reply:
Added